### PR TITLE
feat: remove side padding in bacnk link

### PR DESCRIPTION
**Pull request by chriscorrales**
# Pull Request Template

## Description

Remove side padding to align link left

## Screenshots (if applicable)
image

### CODE REVIEW

**Review by farisfaikar — CHANGES_REQUESTED** [review]
Tl;dr: just add `px-0` class to the back button. As simple as that. No need to overengineer stuff

File: components/count-sentence.tsx
Comment (farisfaikar) [inline components/count-sentence.tsx:1]: I'm unsure why you use a custom `noSidePadding` size here. It doesn't make sense, frankly. A simple fix would be to add `px-0` to the class as such:

image

File: components/ui/button.tsx
Comment (farisfaikar) [inline components/ui/button.tsx:1]: Customizing the `button` component like this is overkill. `noSidePadding` doesn't sound like it belongs in the size object. Please refrain from changing the `button` component unless the circumstance absolutely requires it. In this case, a simple `px-0` to override an instance of the button component is enough to do the job.

**Review by farisfaikar — CHANGES_REQUESTED** [review]
Tl;dr: just add `px-0` class to the back button. As simple as that. No need to overengineer stuff

File: components/ui/button.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ const buttonVariants = cva(
       },
       size: {
         default: "h-10 px-4 py-2",
+        noSidePadding: "h-10 py-2",
```
Comment (farisfaikar) [inline components/ui/button.tsx:21]: Customizing the `button` component like this is overkill. `noSidePadding` doesn't sound like it belongs in the size object. Please refrain from changing the `button` component unless the circumstance absolutely requires it. In this case, a simple `px-0` to override an instance of the button component is enough to do the job.

File: components/count-sentence.tsx [hunk 2]
```diff
@@ -200,7 +200,7 @@ export default function CountSentence({ className = "" }: CountSentenceProps) {
           )}
         </Button>
       </div>
-      <Button variant="link" className="mt-5 flex gap-2" onClick={goToHome}>
+      <Button variant="link" size="noSidePadding" className="mt-5 flex gap-2" onClick={goToHome}>
```
Comment (farisfaikar) [inline components/count-sentence.tsx:203]: I'm unsure why you use a custom `noSidePadding` size here. It doesn't make sense, frankly. A simple fix would be to add `px-0` to the class as such:

image

**Review by farisfaikar — CHANGES_REQUESTED** [review]
Tl;dr: just add `px-0` class to the back button. As simple as that. No need to overengineer stuff

**Review by farisfaikar — APPROVED** [review]
Looks good. Thanks for contributing!
